site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
++++
+date = "2018-09-10T12:21:53-05:00"
+title = "Citra Progress Report - 2018 Q2"
+tags = [ "progress-report" ]
+author = "anodium"
+coauthor = "CaptV0rt3x"
+forum = 49673
++++
+
+Hello everyone to the summer (or winter, for those on the southern half of this
+Earth) edition of Citra's progress reports. Although not very many large changes
+have been made, many fixes have focused on accuracy improvements and implementing
+the sorely needed user experience features.
+
+Before we go further into this, our [patreon](https://www.patreon.com/citraemu/posts)
+poll results are out.
+For those of you who didn't know, we conducted a poll on our patreon page,
```

Comment:
`Patreon`. Further, don't like referral to the reader in this context - just jump into `We asked Patreons which feature [...]` maybe?